content/security/reporting.adoc

```diff
@@ -95,6 +95,7 @@ The following behaviors/issues are not vulnerabilities in Jenkins project infras
   We trust Google to choose the correct tradeoffs for their Google Groups service, and have no power to change anything anyway.
 * The public availability of the Algolia API key. It needs to be public.
 // TODO Possibly other keys as well?
+* Publicly accessible Jenkins instances other than ci.jenkins.io and weekly.ci.jenkins.io are not operated by the Jenkins project. Please do not contact us about any concerns about them.
```

Comment:
```suggestion
* Publicly accessible Jenkins instances other than ci.jenkins.io and weekly.ci.jenkins.io are not operated by the Jenkins project.
  Please do not contact us about any concerns about them.
```

The rest of this file has sentence-per-line formatting, please do the same here for consistency.

Also, possibly ungrammatical?